src/App.css

```diff
@@ -90,6 +90,21 @@
   margin: 10px 0;
 }
 
+.single-article {
```

Comment:
Love the styling that's in place already 👍 and you have a bottom navigation bar too! This is very cool.